lib/bootstrap.js

```diff
@@ -145,7 +145,11 @@ const bootstrap = {
         // TODO: evaluate if `wrap` is actually needed?
         yargs.wrap(Math.min(100, yargs.terminalWidth()))
             .epilogue('For more information, see our docs at https://docs.ghost.org/v1/docs/ghost-cli')
-            .option('D', {
+            .option('d', {
+                alias: 'dir',
+                describe: 'Folder to run command in',
+                default: './'
```

Comment:
Not sure if we should have a default value - I think setting it to `type: 'string'` and then only executing the code in `command.js` when `argv.dir` is defined is maybe a better approach? Not sure what the performance implications of calling `process.chdir` are, but it might be better to not call it if it's not needed 😄